### PR TITLE
[DO NOT MERGE] Improve Apify dataset source docs

### PR DESCRIPTION
**Pull request by mhamas**
This PR is opened in order to get feedback on Apify dataset source docs in Airbyte. Once polished, I will open PR against the parent repo.

### CODE REVIEW

File: docs/integrations/sources/apify-dataset.md [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+[Apify](https://www.apify.com) is a web scraping and web automation platform providing ready and custom made solutions, [SDK](https://sdk.apify.com/),
+and other tools, such as proxies, to build and run any job at scale. 
+
+The results of a scraping job are usually stored in [Apify Dataset](https://docs.apify.com/storage/dataset). This Airbyte connector allows you
+to automatically sync the contents of a dataset to your chosen destination using Airbyte.
+
+To sync data from a dataset, all you need to know is its ID. You will find it in [Apify console](https://my.apify.com/) under storages.
 
-The Apify dataset connect supports full refresh sync only.
+### Running Airbyte sync from Apify webhook
```
Comment (mtrunkat) [inline docs/integrations/sources/apify-dataset.md:16]: Do we need to put together a POST request body payload? If so then let's add an example

**Review by mtrunkat — CHANGES_REQUESTED** [review]

File: docs/integrations/sources/apify-dataset.md (hunk 1 above)
Comment (mhamas) [reply docs/integrations/sources/apify-dataset.md:16]: Adding this screenshot
CleanShot 2021-08-16 at 11 11 28@2x

**Review by mtrunkat — APPROVED** [review]

File: docs/integrations/sources/apify-dataset.md [hunk 2]
```diff
@@ -1,11 +1,27 @@
+---
+description: >-
+  Web scraping and automation platform.
+---
+
 # Apify dataset
 
 ## Overview
 
+[Apify](https://www.apify.com) is a web scraping and web automation platform providing ready and custom made solutions, [SDK](https://sdk.apify.com/),
```
Comment (jancurn) [inline docs/integrations/sources/apify-dataset.md]: How about:

[Apify](https://apify.com) is a web scraping and web automation platform providing both ready-made and custom solutions, an open-source [SDK](https://sdk.apify.com/) for web scraping, [proxies](https://apify.com/proxies), and many other tools to help you build and run web automation jobs at scale.

**Review by jancurn — APPROVED** [review]

**Comment by mhamas** [comment]
Reopened against upstream https://github.com/airbytehq/airbyte/pull/5452

**Pull request closed without merging**